susemanager-utils/susemanager-sls/src/modules/mgractionchains.py

```diff
@@ -160,6 +156,11 @@ def next(actionchain_id, chunk, next_action_id=None, ssh_extra_filerefs=None):
         yaml_dict['ssh_extra_filerefs'] = ssh_extra_filerefs
     _persist_next_ac_chunk(yaml_dict)
 
+def reboot(actionchain_id, chunk, next_action_id=None, ssh_extra_filerefs=None):
+    ret = __salt__['rebootmgr.reboot']()
+    next(actionchain_id, chunk, next_action_id, ssh_extra_filerefs)
```

Comment:
A bit off-topic: when reading this, I thought "that does not look right, `next` requires an iterator and is not called like that". Then I realized this is not the [built-in `next()`](https://docs.python.org/3/library/functions.html#next), and we shadow its name. We should address that, but that can be in a different PR.